src/components/common/Uploader.js

```diff
@@ -49,6 +51,32 @@ class Uploader extends Component {
       autoProceed: true,
     });
 
+    // when offline override upload to post corresponding resources
```

Comment:
How does this actually work?

Reply:
uppy implements its own `upload` function which actually uploads online the file (given the options when initialized). To avoid this from happening, I override the `upload` function where I dispatch `postAppInstanceResource` and fake the upload success to avoid uppy to throw an error

Reply:
OK. Sounds good. Do we handle the failure?